quickjs.c

```diff
@@ -14557,20 +14557,35 @@ static BOOL js_strict_eq(JSContext *ctx, JSValue op1, JSValue op2)
     return js_strict_eq2(ctx, op1, op2, JS_EQ_STRICT);
 }
 
+BOOL JS_StrictEq(JSContext *ctx, JSValue op1, JSValue op2)
+{
+    return js_strict_eq(ctx, op1, op2);
+}
+
```

Comment:
For consistency, this API should also take 2 `JSValueConst` operands.

Suggestion:
```
BOOL JS_StrictEq(JSContext *ctx, JSValueConst op1, JSValueConst op2)
{
    return js_strict_eq(ctx,  JS_DupValue(ctx, op1), JS_DupValue(ctx, op2));
}
```